src/backend/TrafficCourts/Workflow.Service/Consumers/DisputantUpdateRequestConsumer.cs

```diff
@@ -0,0 +1,76 @@
+﻿
+
+using MassTransit;
+using System.Text.Json;
+using TrafficCourts.Common.OpenAPIs.OracleDataApi.v1_0;
+using TrafficCourts.Workflow.Service.Services;
+using DisputantUpdateRequest = TrafficCourts.Messaging.MessageContracts.DisputantUpdateRequest;
+
+namespace TrafficCourts.Workflow.Service.Consumers;
+
+public class DisputantUpdateRequestConsumer : IConsumer<DisputantUpdateRequest>
+{
+    private readonly ILogger<DisputantUpdateRequestConsumer> _logger;
+    private readonly IOracleDataApiService _oracleDataApiService;
+
+    public DisputantUpdateRequestConsumer(ILogger<DisputantUpdateRequestConsumer> logger, IOracleDataApiService oracleDataApiService)
+    {
+        _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        _oracleDataApiService = oracleDataApiService ?? throw new ArgumentNullException(nameof(oracleDataApiService));
+    }
+
+    public async Task Consume(ConsumeContext<DisputantUpdateRequest> context)
+    {
+        _logger.LogDebug("Consuming message");
+        DisputantUpdateRequest message = context.Message;
+
+        Common.OpenAPIs.OracleDataApi.v1_0.DisputantUpdateRequest disputantUpdateRequest = new()
+        {
+            Status = DisputantUpdateRequestStatus2.PENDING,
+            UpdateJson = JsonSerializer.Serialize(message)
+        };
+
+        if (message.EmailAddress is not null)
+        {
+            // Set status to HOLD. This status will be used if there are any subsequent name, address, or phone update requests
+            // The status will move from HOLD to PENDING when the disputant's email has been verified.
+            disputantUpdateRequest.Status = DisputantUpdateRequestStatus2.HOLD;
+
+            // TODO: Start email saga. TCVP-2009

```

Comment:
Placeholder to invoke the email saga (TCVP-2009)

Reply:
This looks great but I think we need to change the flow a bit. We need to publish two events in case there is an email update. One is the existing `RequestEmailVerification` event which triggers the existing Saga for email verification and the second one will be `DisputantUpdateRequestSubmitted` event which will trigger the new Saga with the new unique ids of the inserted `DisputantUpdateRequest` records and an `UpdateRequestId`. We can discuss this in detail.